docker/Dockerfile.base.16.04

```diff
@@ -101,7 +101,8 @@ RUN pip install --upgrade pip
 # In this step only the presence of the file mjkey.txt is required, so we only
 # create an empty file
 RUN touch /root/.mujoco/mjkey.txt && \
-  pip install .[all,dev] && \
+  pip install .[all] && \
```

Comment:
please only change things which absolutely need to be changed to make this work. AFAICT we only need to do this in `ci-verify-conda` and nowhere else.

Reply:
This change is also necessary, or baselines will fail to install because tensorflow wasn't installed first. I'm going to copy it to my change.